### DISCUSSION TITLE
006

### DISCUSSION THREAD

**Discussion by giscus** [zhiyu1998/rrorange-and-friends-weekly#4] [General]
# 006

《大橘和朋友们的周刊》： 分享日常冲浪互联网看到好玩的网站、app应用、资源分享、效率软件工具集等

https://rrorangeandfriends.site/posts/2024/006

**Comment by loveliveao** [comment]
大橘，Spotify的卡片R角外面有一圈白色的像素

**Reply by zhiyu1998** [reply]
这个俺可能也解决不了
